### PR TITLE
resolve aur dependencies

### PR DESCRIPTION
**Pull request by koehlma**
If a package has some dependencies that are only available in the AUR ask the user if he wants to add them to the local repository and install them. Only works if the local repository is available in the system.

Example: `local-repo repo -A yaourt` asks to add `package-query` to the local repository.

### CODE REVIEW

**Comment by ushis** [comment]
Hey @koehlma 

I don't have the time to maintain this project, but your pull request looks good to me and i'm willing to give you commit access to this repo.

Ping me, if you are interested.

P.S. Please use the devel branch as merge target.

File: localrepo/pacman.py [hunk 1]
```diff
@@ -84,8 +97,8 @@ def uninstall(pkgs):
 		for i, pkg in enumerate(pkgs):
 			pkgs[i] = Pacman.VERSION_SEP.split(pkg)[0]
 
-		Pacman._run_as_root([Pacman.PACMAN, '-Rs'] + list(set(pkgs)))
-
+		Pacman._run_as_root([Pacman.PACMAN, '-Rsc'] + list(set(pkgs)))
```
Comment (ushis) [inline localrepo/pacman.py]: Why is this necessary? I think it's kinda dangerous here.

Comment (koehlma) [inline localrepo/pacman.py]: This was necessary because otherwise I got an error when trying to uninstall the dependencies because an other package requires them but I did not explicit install this package - I tried this in a fresh chroot environment. But maybe I just screwed up the database with my testing. Tomorrow I am going to test this again.

**Comment by koehlma** [comment]
Hi,

if you want to, then yes, I am interested but my time is also limited.

P.S. I first wanted to use the devel branch but then I noticed that the master branch is actually one commit ahead  - so I decided to use the master branch.

**Comment by ushis** [comment]
> P.S. I first wanted to use the devel branch but then I noticed that the master branch is actually one commit ahead - so I decided to use the master branch.

Ok, no problem, `devel` is up to date now.
